docs/guide/validation.md

```diff
@@ -15,69 +15,153 @@ You can find the detailed descriptions of the _validations_ used in the fields c
 * [Detailed docs](https://pkg.go.dev/github.com/go-playground/validator?tab=doc)
 
 ```go title="Validation Example"
-type Job struct{
-    Type          string `validate:"required,min=3,max=32"`
-    Salary        int    `validate:"required,number"`
-}
+package main
 
-type User struct{
-    Name          string  `validate:"required,min=3,max=32"`
-    // use `*bool` here otherwise the validation will fail for `false` values
-    // Ref: https://github.com/go-playground/validator/issues/319#issuecomment-339222389
-    IsActive      *bool   `validate:"required"`
-    Email         string  `validate:"required,email,min=6,max=32"`
-    Job           Job     `validate:"dive"`
-}
+import (
+	"fmt"
+	"log"
+	"strings"
 
-type ErrorResponse struct {
-    FailedField string
-    Tag         string
-    Value       string
-}
+	"github.com/go-playground/validator"
```

Comment:
can you change this one with v10

Reply:
@Z3NTL3

Reply:
nice catch @efectn

Reply:
Fixed it, look at <a href="https://github.com/gofiber/fiber/pull/2517/commits/34cffe582891693b83347708f07c6f7fb601edd3">commit</a>